### PR TITLE
Upgraded JUnit to 5.3.2

### PR DESCRIPTION
**Pull request by AbhayChandel**

### CODE REVIEW

**Comment by maybeec** [comment]
@hohwille please see #136, which contains our upgrade experiences from my project.

**Comment by hohwille** [comment]
> please see #136, which contains our upgrade experiences from my project.

I did but could not find anything regarding JUnit at all. Do you mean something else? Like some meta-information that we are updating to much and projects cannot cope? Or what am I missing?

File: templates/server/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -79,6 +84,7 @@
 
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.22.0</version>
         <configuration>
           <excludedGroups>${devonfw.test.excluded.groups}</excludedGroups>
```
Comment (hohwille) [inline templates/server/src/main/resources/archetype-resources/pom.xml]: The categories have changed to tags with different IDs. I am missing the diff to update these:
https://github.com/AbhayChandel/devon4j/blob/issue-130-update-JUnit-to-5.x/templates/server/src/main/resources/archetype-resources/pom.xml#L18

Does that still work with `excludedGroups` attributes after switching to tags?

**Comment by hohwille** [comment]
Replaced by PR #151

**Pull request closed without merging**